File: app/src/main/java/com/sysu/edu/academic/HomeworkActivity.java
package com.sysu.edu.academic;

import android.os.Bundle;

import androidx.navigation.NavController;
import androidx.navigation.fragment.NavHostFragment;
import androidx.navigation.ui.NavigationUI;

import com.sysu.edu.BaseActivity;
import com.sysu.edu.R;
import com.sysu.edu.databinding.ActivityHomeWorkBinding;

public class HomeworkActivity extends BaseActivity {
    
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityHomeWorkBinding binding = ActivityHomeWorkBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        NavHostFragment navHostFragment = (NavHostFragment) getSupportFragmentManager().findFragmentById(R.id.fragment);
        if (navHostFragment != null) {
            NavController navController = navHostFragment.getNavController();
            NavigationUI.setupWithNavController(binding.bottomNav, navController);
        }
        binding.toolbar.setNavigationOnClickListener(_ -> supportFinishAfterTransition());
    }
}

File: app/src/main/java/com/sysu/edu/academic/HomeworkActivity.kt
package com.sysu.edu.academic

import android.os.Bundle
import androidx.navigation.fragment.NavHostFragment
import androidx.navigation.ui.NavigationUI.setupWithNavController
import com.sysu.edu.BaseActivity
import com.sysu.edu.R
import com.sysu.edu.databinding.ActivityHomeWorkBinding

class HomeworkActivity : BaseActivity() {
	override fun onCreate(savedInstanceState: Bundle?) {
		super.onCreate(savedInstanceState)
		val binding = ActivityHomeWorkBinding.inflate(layoutInflater)
		setContentView(binding.getRoot())
		val navHostFragment = supportFragmentManager.findFragmentById(R.id.fragment) as NavHostFragment
		 setupWithNavController(binding.bottomNav, navHostFragment.navController)
		binding.toolbar.setNavigationOnClickListener { supportFinishAfterTransition() }
	}
}